### PR TITLE
Upgrade update-dependencies

### PR DESCRIPTION
**Pull request by eerhardt**
Also, updating to SharedFramework 2.0.0-beta-001509-00 testing out the changes.

/cc @dotnet/dotnet-cli

### CODE REVIEW

**Comment by eerhardt** [comment]
@dagood - to checkout the update dependencies changes.

File: build_projects/update-dependencies/Program.cs [hunk 1]
```diff
@@ -21,75 +21,57 @@ public static void Main(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
+            bool onlyUpdate = args.Length > 0 && string.Equals("--Update", args[0], StringComparison.OrdinalIgnoreCase);
```
Comment (dagood) [inline build_projects/update-dependencies/Program.cs]: [nit] Seems like a weird way to check for this arg, but I see it was just moved from below.

File: build_projects/update-dependencies/Program.cs [hunk 2]
```diff
@@ -21,75 +21,57 @@ public static void Main(string[] args)
...
-                s_config.GitHubProject,
-                s_config.GitHubUpstreamOwner,
-                s_config.GitHubUpstreamBranch,
-                s_config.UserName,
-                s_config.GitHubPullRequestNotifications);
-
-            if (args.Length > 0 && string.Equals("--Update", args[0], StringComparison.OrdinalIgnoreCase))
+            var dependencyBuildInfos = buildInfos.Select(i =>
+                new DependencyBuildInfo(
+                    i,
```
Comment (dagood) [inline build_projects/update-dependencies/Program.cs]: [nit: naming] I'd expect `i` to be the index (for an `(info, i)` `Select`).

Comment (eerhardt) [inline build_projects/update-dependencies/Program.cs]: renamed to 'buildInfo'

File: build_projects/update-dependencies/Program.cs [hunk 3]
```diff
@@ -21,75 +21,57 @@ public static void Main(string[] args)
...
-                dependencyUpdater.UpdateAndSubmitPullRequestAsync(updaters, buildInfos);
+                GitHubAuth gitHubAuth = new GitHubAuth(s_config.Password, s_config.UserName, s_config.Email);
+                GitHubProject origin = new GitHubProject(s_config.GitHubProject, s_config.UserName);
+                GitHubBranch upstreamBranch = new GitHubBranch(
+                    s_config.GitHubUpstreamBranch,
+                    new GitHubProject(s_config.GitHubProject, s_config.GitHubUpstreamOwner));
+
+                string suggestedMessage = updateResults.GetSuggestedCommitMessage();
+                string body = string.Empty;
+                if (s_config.GitHubPullRequestNotifications != null)
```
Comment (eerhardt) [inline build_projects/update-dependencies/Program.cs]: I need to check for `Any()` here as well.

**Pull request merged**